scss/elements/_typography.scss

```diff
@@ -136,7 +136,7 @@ dfn {
   }
 
   > :first-child {
-    @include media($medium-screen) {
+    @media #{$breakpoint-md} {
```

Comment:
I'd rather see this use neat's grid for consistency rather than a raw mediaquery.

Reply:
this is an existing simple media query, so I needed to do this in order to avoid compilation errors.